src/tqec/plaquette/plaquette.py

```diff
@@ -26,15 +30,18 @@ def __init__(self, qubits: list[PlaquetteQubit], circuit: ScheduledCircuit) -> N
             coordinate system.
         :param circuit: scheduled quantum circuit implementing the computation that the
             plaquette should represent.
+
+        :raises ValueError: if the number of qubits doesn't match the number of qubits
         """
+        # if len(qubits) != len(circuit.raw_circuit.all_qubits()):
+        #     raise ValueError("Number of qubits doesn't match number of circuit qubits")
+
```

Comment:
Either the commented code should be uncommented or the docstring part about `:raises:` should be removed.

Reply:
I was wondering why this wasn't working, but I don't have the time atm. to look into it.

Reply:
Turns out it is because the `cirq.Circuit` only considers the qubits on which gates are applied and some `ScheduledCircuit` instances have `cirq.Circuit` instances with unused qubits.
I found a way to make that work, the check was a good idea. I will propose patches.